hs_core/irods.py

```diff
@@ -156,27 +165,28 @@ def list_ticket(self, ticket):
                     value = line[1]
                     if key == 'collection name' or key == 'data collection':
                         output['full_path'] = value
-                        if self.resource_federation_path:
+                        if self.is_federated:
                             if __debug__:
                                 assert(value.startswith(self.resource_federation_path))
                             output['long_path'] = value[len(self.resource_federation_path):]
-                            output['fed_path'] = self.resource_federation_path
+                            output['home_path'] = self.resource_federation_path
                         else:
                             location = value.search(self.short_id)
                             if __debug__:
                                 assert(location >= 0)
-                            output['long_path'] = value[location:]
-                            output['home_path'] = value[:(location-1)]
+                            if location == 0:
+                                output['long_path'] = value
+                                output['home_path'] = self.irods_home_path
+                            else:
+                                output['long_path'] = value[location:]
+                                output['home_path'] = value[:(location-1)]
```

Comment:
It is better documented then and I assume the trailing slash will always be present from ticket listing output.